scripts/galera_new_cluster.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+
+VERSION="@VERSION@@MYSQL_SERVER_SUFFIX@"
+COMPILATION_COMMENT="@COMPILATION_COMMENT@"
+
+/usr/bin/systemctl set-environment MYSQLD_OPTIONS='--wsrep-new-cluster' && \
+     /usr/bin/systemctl start mariadb.service
+
+/usr/bin/systemctl set-environment MYSQLD_OPTIONS=''
```

Comment:
I was wondering how to do this. There isn't a good systemctl get-environment interface so I'll add a new variable $WSREP_NEW_CLUSTER to the service.in files and use that in the script. I'll also add an optional script argument of the service name defaulting to mariadb.service.